examples/macos/itunes_example.py

```diff
@@ -0,0 +1,67 @@
+# GUI Application automation and testing library
```

Comment:
Please rename file `examples/macos/itunes_example.py` to `examples/macos/itunes.py`. Folder `examples` already tells user what it is.